Dockerfile

```diff
@@ -22,9 +22,10 @@ COPY --from=0 /src/packages/server/cli-linux-x64 /usr/local/bin/code-server
 EXPOSE 8443
 RUN apt-get update && apt-get install -y \
 	openssl \
-	net-tools
-RUN apt-get install -y locales && \
-	locale-gen en_US.UTF-8
+	net-tools \
+	locales && \
+	locale-gen en_US.UTF-8 && \
+	rm -rf /var/lib/apt/lists/*
```

Comment:
you forgot to run `apt-get clean` first.

Reply:
``Official Debian and Ubuntu images automatically run apt-get clean, so explicit invocation is not required.``
Source: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/